modules/ezcontent_paragraphs/modules/ezcontent_card/ezcontent_card.info.yml

```diff
@@ -4,6 +4,7 @@ description: 'Provides card feature for promoting content.'
 package: EZContent Paragraphs
 core: 8.x
 dependencies:
+  - drupal:color_field
```

Comment:
This dependency would go into module than parent module.
Also, `drupal:color_field` is incorrect. color_field is not a Drupal Core module.